### PR TITLE
Remove name

### PR DESCRIPTION
**Pull request by madelinecameron**
re: discussion wells and I had. 

Thoughts?

### CODE REVIEW

**Comment by jwerle** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -18,23 +18,26 @@ const Persist = require('persist-store')
...
+    return Promise.resolve('shh, act like i loaded something')
   }
 }
 
 // example options
 const persist = new Persist([
     {
       type: 'local',
-      path: './data'
+      path: './data',
```
Comment (wellsjo) [inline README.md]: why not take the full path to file?

File: README.md [hunk 2]
```diff
@@ -18,23 +18,26 @@ const Persist = require('persist-store')
...
+      name: 'file'
     },
     {
       type: 's3',
       bucket: 'a_bucket',
+      prefix: 'some_prefix',
```
Comment (wellsjo) [inline README.md]: same thing here, just take bucket and key

File: README.md (hunk 1 above)
Comment (madelinecameron) [reply README.md]: I was doing that originally and I cooould do that, but the big issue is that if the directory doesn't exist, it creates it. 

I know I could split it and all and but eh, figured more deliberate was better. Either is cool with me.

File: README.md (hunk 2 above)
Comment (madelinecameron) [reply README.md]: Ok, I just thought bucket described the top level then prefix was middle level and key was bottom level. I'm cool with this, I'll change it.  (I know it doesn't matter to S3, just thought it was how to describe the terms :))

File: README.md [hunk 3]
```diff
@@ -18,23 +18,24 @@ const Persist = require('persist-store')
...
 const persist = new Persist([
     {
       type: 'local',
-      path: './data'
+      path: './data/file',
     },
     {
       type: 's3',
-      bucket: 'a_bucket',
+      bucket: 'a_bucket/some_prefix',
```
Comment (wellsjo) [inline README.md]: bucket should just be bucket - s3 doesn't really have the concept of prefixes as it's just a key/value store under the hood. you can think of `bucket` as its own filesystem and `key` as the full path to the file. 

"prefix" is only used for listing multiple keys in s3 (that have the same prefix), and isn't actually part of the architecture

File: lib/persisters/local.js [hunk 4]
```diff
@@ -9,41 +9,27 @@ const fs = require('fs')
...
     if (!opts.path) {
       throw new Error('`path` is undefined for local persister')
     }
 
-    const directory = path.resolve(__dirname, opts.path)
+    const directory = path.resolve(__dirname, opts.path.split('/').slice(0, -1).join('/'))
     mkdirp(directory)
 
-    this.directory = directory
+    this.directory = opts.path
```
Comment (wellsjo) [inline lib/persisters/local.js]: seems like you're assigning a file path to `this.directory` which is kind of confusing. why not `this.path`?

**Pull request merged**